src/Components/Facility/CentralNursingStation.tsx

```diff
@@ -70,6 +74,26 @@ export default function CentralNursingStation({ facilityId }: Props) {
     "3xl": 12 / 11,
   });
 
+  useEffect(() => {
+    const updatePerPageLimit = () => {
+      if (window.innerWidth > LARGE_SCREEN_WIDTH) {
+        console.log("hello");
+        setPerPageLimit(PER_PAGE_LIMIT_LARGE);
+      } else {
+        console.log("hii");
+        setPerPageLimit(PER_PAGE_LIMIT);
+      }
+    };
+
+    updatePerPageLimit();
+
+    window.addEventListener("resize", updatePerPageLimit);
+
+    return () => {
+      window.removeEventListener("resize", updatePerPageLimit);
+    };
+  }, []);
```

Comment:
checkout `useBreakpoints` custom hook. We could use that instead of rewriting this logic everywhere.

Reply:
Sure, @rithviknishad, I'll implement the modifications.

Reply:
@rithviknishad I have made the changes please review it